docs/DeveloperGuide.adoc

```diff
@@ -844,25 +899,156 @@ Use case ends.
 +
 Use case resumes at step 2.
 
+=== Use case: Add an order
+
+**MSS**
+
+1.  User request to add an order
+2.  Delivery adds the offer to the current queue
```

Comment:
Write in a forth-back manner. Try to make it A-B-A-B instead of A-B-B. This is a format issue. Avoid typos like missing "s".